source/standard/SIDS/grid.rst

```diff
@@ -450,14 +450,16 @@ The third and fourth examples are for an unstructured zone with 15 tetrahedral a
 Example - TETRA_4 Element Types
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This example uses the simple three-element tetrahedral grid shown below.
+This example uses a simple three-element tetrahedral grid.
 
-.. figure:: ../../../images/sids/figs/ex_tetra.gif
-   :width: 400px
-   :align: center
-   :alt: Unstructured grid consisting of three tetrahedra
-
-   *Example Tetrahedral Grid*
+.. .. figure:: ../../../images/sids/figs/ex_tetra.gif
+..    :width: 400px
+..    :align: center
+..    :alt: Unstructured grid consisting of three tetrahedra
+..
+..    *Example Tetrahedral Grid*
+..
+.. (Figure pending - ex_tetra.gif not found)

```

Comment:
The missing file exists here : https://github.com/CGNS/cgns-archives/blob/develop/CGNS_docs_current/sids/gridflow.figs/ex_tetra.gif